.appveyor.yml

```diff
@@ -1,6 +0,0 @@
-image: Visual Studio 2017
-configuration:
-  - Release
-  - Debug
-build:
-  project: OP2Helper.sln
```

Comment:
After merging master, and then removing the `.appveyor.yml` file on this branch, this would cause cause `.appveyor.yml` to be removed from master when this branch is merged to master. I assume that wasn't quite the intention here.

As for getting AppVeyor builds working, the project still needs to be added on the AppVeyor website. Adding the project there installs any needed hooks on GitHub, so AppVeyor will be informed when a new commit is pushed. I just added the project over on AppVeyor.